test/functionality/_helper/shell.ts

```diff
@@ -447,60 +447,76 @@ function testWrapper(skip: boolean | undefined, shouldFail: boolean, testName: s
 
 export type TestCaseFailType = 'fail-shell' | 'fail-tree-sitter' | 'fail-both' | undefined;
```

Comment:
Not to become too yak shavy, but we only seem to use `TestCaseFailType` `fail-both`. Do you suspect that we will ever need the other two cases or could we simplify this to a boolean, @EagleoutIce?